package.json

```diff
@@ -85,15 +85,13 @@
     "standard-version": "^9.5.0",
     "ts-node": "^10.9.1",
     "tslib": "^2.3.0",
-    "typescript": "~4.7.2",
+    "typescript": "~4.9.5",
     "zone.js": "~0.11.4"
   },
   "dependencies": {
     "tslib": "^2.3.0"
   },
   "peerDependencies": {
-    "@angular/core": ">=5.0.0",
```

Comment:
Please re-add these peer dependencies and set them to `>=15.0.0` as they can be quick helpful to users